grpc-gateway/service/getResourceFromDevice_test.go

```diff
@@ -165,3 +168,105 @@ func TestRequestHandlerGetResourceFromDevice(t *testing.T) {
 		})
 	}
 }
+
+func validateETags(ctx context.Context, t *testing.T, c pb.GrpcGatewayClient, deviceID, href string) {
+	sdkClient, err := test.NewSDKClient()
+	require.NoError(t, err)
+
+	defer func() {
+		err := sdkClient.Close(context.Background())
+		require.NoError(t, err)
+	}()
+
+	cfg1 := coap.DetailedResponse[interface{}]{}
+	err = sdkClient.GetResource(ctx, deviceID, href, &cfg1)
+	require.NoError(t, err)
+
+	cfg2, err := c.GetResourceFromDevice(ctx, &pb.GetResourceFromDeviceRequest{
+		ResourceId: commands.NewResourceID(deviceID, href),
+		TimeToLive: int64(time.Hour),
+	})
+	require.NoError(t, err)
+	require.Equal(t, cfg1.ETag, cfg2.GetData().GetEtag())
+	var body2 interface{}
+	err = cbor.Decode(cfg2.GetData().GetContent().GetData(), &body2)
+	require.NoError(t, err)
+	require.Equal(t, cfg1.Body, body2)
+	/*
+		TODO: uncomment this block when etag will be supported in by coap notifications
```

Comment:
Won't this work with the latest go-coap or are additional changes required?

Reply:
No because notifications doesn't provides etags